routers/web/repo/projects.go

```diff
@@ -161,14 +161,15 @@ func NewProjectPost(ctx *context.Context) {
 
 // ChangeProjectStatus updates the status of a project between "open" and "close"
 func ChangeProjectStatus(ctx *context.Context) {
-	toClose := false
+	var toClose bool
 	switch ctx.Params(":action") {
 	case "open":
 		toClose = false
 	case "close":
 		toClose = true
 	default:
 		ctx.Redirect(ctx.Repo.RepoLink + "/projects")
```

Comment:
Should nt be this also changed @lng2020 ?

Reply:
Reasonable. Changed.